### ISSUE TITLE
Seedlet: Custom link colors not rendered on frontend

### ISSUE THREAD

**Issue by sirreal** [Automattic/wp-calypso#45456]
#### Steps to reproduce

1. Activate [Seedlet](https://wordpress.com/theme/seedlet) on a Simple site
1. Open the block editor and write a paragraph with a link.
1. From the block controls, expand color settings and change the link color.
1. Publish the post and view it on the site (example post: paqfU8-tn-p2)
1. The link color will not be rendered on the frontent.

Note that custom background and text colors (rendered as `background-color` and `color` in a `style` attribute) are respected. The custom link color is not.

#### What I expected

Render the link color.

#### What happened instead

The link color is not respected.

#### Screenshot / Video

**It _says_ "green" but the link _should be `#ff00ff` "pink".**

##### Editor
![Screen Shot 2020-09-08 at 12 53 00](https://user-images.githubusercontent.com/841763/92467663-46b64d00-f1d2-11ea-9edb-a79b5b7b778c.png)
![Screen Shot 2020-09-08 at 12 52 36](https://user-images.githubusercontent.com/841763/92467671-47e77a00-f1d2-11ea-8fd8-7a004ed869c3.png)

##### Frontend

![Screen Shot 2020-09-08 at 12 51 33](https://user-images.githubusercontent.com/841763/92467657-41f19900-f1d2-11ea-80fd-bcf89eb60572.png)

![Screen Shot 2020-09-08 at 12 51 44](https://user-images.githubusercontent.com/841763/92467655-4027d580-f1d2-11ea-821d-ae2ba5627021.png)

**Comment by nosolosw** [comment]
I could repro this issue in a dotorg setup with an author role, see https://github.com/WordPress/gutenberg/issues/25151 In dotcom, I experienced this by testing with an admin role.

**Comment by nosolosw** [comment]
@sirreal @scruffian Got a potential fix for this at https://github.com/WordPress/gutenberg/pull/25411 Note, however, what I reported above: it looks like dotcom may be doing additional things to what core does, so it's worth looking into that separately.

**Comment by nosolosw** [comment]
Ah, it'd be great if you have a way to report whether the fix works for dotcom as it is.

**Comment by WunderBart** [comment]
> Ah, it'd be great if you have a way to report whether the fix works for dotcom as it is

@nosolosw, I installed the build from https://github.com/WordPress/gutenberg/pull/25411 in D50012-code & tested in on my side but it doesn't seem that this patch alone fixes the issue:

| Editor  | Preview |
| ------------- | ------------- |
| <img width="293" alt="Screenshot 2020-09-22 at 15 46 59" src="https://user-images.githubusercontent.com/1451471/93911793-25da1580-fd03-11ea-8437-a854f9d5a37a.png"> | <img width="287" alt="Screenshot 2020-09-22 at 15 47 09" src="https://user-images.githubusercontent.com/1451471/93911796-2672ac00-fd03-11ea-9089-4007654d666f.png"> |

**Comment by WunderBart** [comment]
@nosolosw has there been any follow-up regarding this issue on dotcom? This seems to be still happening.

**Comment by nosolosw** [comment]
I can't speak for dotcom but, for core, we're going to revisit how to fix this (see [comment](https://github.com/WordPress/gutenberg/pull/25411#issuecomment-698401806)).

**Comment by sirreal** [comment]
> has there been any follow-up regarding this issue on dotcom? This seems to be still happening.

Context: There was a Seedlet theme update, which is why we revisited 🙂

**Comment by WunderBart** [comment]
Let's revisit this when 9.2 lands.

**Comment by andrewserong** [comment]
Just confirming that this is still an issue in wpcom with Seedlet theme activated.

In the editor, with a custom link color set (to red):

![image](https://user-images.githubusercontent.com/14988353/100027600-37de4e80-2e41-11eb-8dae-21dc235a5718.png)

In the front end:

![image](https://user-images.githubusercontent.com/14988353/100027631-49bff180-2e41-11eb-9bd6-a905b7ab639e.png)

**Comment by scruffian** [comment]
I don't believe this is a theme issue, so I'm going to remove it from our board. If you disagree let me know!

**Comment by Robertght** [comment]
I think this is the same as this one: https://github.com/Automattic/wp-calypso/issues/54990

I'm marking this as a duplicate now in favor of the above.
